docs/source/release-history.rst

```diff
@@ -2,6 +2,15 @@
 Release History
 ***************
 
+v0.1.4 (2021-08-24)
```

Comment:
Should we release it as v0.2.0? The addition of the new xspress3 support may deserve a minor version bump.

Reply:
I almost asked before just going with 0.1.4. I'm happy to release as 0.2.0.

Reply:
Let's do that.